Add support for CIP-0025 version 2 (#594)
input-output-hk/cardano-js-sdk: packages/e2e/README.md

You can configure any of these five wallets in your test and use any amount of tADA you need.

### Local Test Network - development version

> In addition to **local-network** there is its development version, which mounts the `packages` directory in the

Reviewer: ```suggestion
In addition to the **local-network** npm script there is a script suitable for development, which mounts the `packages` directory in
```

Author: af738bc2c should solve this

> http-server container and in the worker container. It helps to apply changes to the two components on a running

Reviewer: ```suggestion
the service containers. It helps to apply changes to the components without shutting down the **local-network** each time.
```

Author: af738bc2c should solve this

**local-network** whithout the need to shutdown and restart it.

It can be started with:

```bash
$ yarn workspace @cardano-sdk/e2e local-network:dev
```

> and a good idea is to run following command before starting it.

Reviewer: I would make this a `prelocal-network:dev` script, then remove this manual step.

```bash
$ yarn cleanup && yarn && yarn build
```

> Once it is running, to restart the http-server it is enough to issue the following command.

Reviewer: ```suggestion
Restart the http-server:
```

Author: af738bc2c should solve this
